client/src/components/Layout/Events/Events.js

```diff
@@ -21,23 +24,50 @@ class Events extends Component {
       .catch((err) => console.log(err))
   }
 
+  changeInput =  (name) => {
+    this.setState({ eventName: name.target.value.substr(0,20)
+       })
+  }
+
   render () {
-    const { events } = this.state
+  const { events, eventName} = this.state
+  const filtertsearch = events.filter(
```

Comment:
pls use camelCase for variables: `filtersearch` => `filterSearch` (also I think there is a typo with the 't' in the middle).

Also, similarily as above, this can be rewritten in just one line:
```js
const filtertsearch = events.filter(event => event.event_title.toLowerCase().indexOf(eventName.toLowerCase()) !== -1);
```